alacritty/src/display/cursor.rs

```diff
@@ -26,7 +26,10 @@ impl IntoRects for RenderableCursor {
             width *= 2.;
         }
 
-        let thickness = (thickness * width as f32).round().max(1.);
+        let mut thickness = (thickness * width as f32).round().max(1.);
+        if self.is_wide() {
+            thickness /= 2.;
+        }
```

Comment:
I feel like there's a little bit of a lack of understanding here when it comes to this issue.

Try moving the line 29 above the `is_wide` check. I'm pretty sure just reordering these statements should resolve this issue.

Alternatively the `cell_width` could be used instead of the `width` variable, to make things a bit more explicit. But I personally think just reusing the existing `width` should be fine.

Reply:
> Try moving the line 29 above the is_wide check. I'm pretty sure just reordering these statements should resolve this issue.

Yes. This is the best way. :+1: